### ISSUE TITLE
statistics: linear regression produces output for `(y,x)` not `(x,y)`.

### ISSUE THREAD

**Issue by a1eaiactaest** [wafflephi/lib#4]
```python
from wafflephi.statistics import LinearRegression
from statistics import linear_regression # python built-in

data = [1.5, 3.8, 6.7, 9, 11.2, 13.6, 16]
est, slope, y_intercept = LinearRegression.lstsq(data)
expected = linear_regression(list(range(len(data)), data)

print(slope, y_intercept, end='\n')
print(expected)

# 0.41377380598450525
# -0.6530316014060611
# LinearRegression(slope=0.4137738059845053, intercept=-0.6530316014060604)
```

**Comment by a1eaiactaest** [comment]
fixed in [cdfe486](https://github.com/wafflephi/lib/commit/cdfe4868d6909df0ffa034dfa68c7bb194804012).

**Comment by a1eaiactaest** [comment]
corrected README example
